dydx/client.py

```diff
@@ -267,101 +293,124 @@ def get_my_fills(
         :raises: DydxAPIError
         '''
         return self.get_fills(
-            pairs=pairs,
-            makerAccountOwner=self.public_address,
-            makerAccountNumber=self.account_number,
+            market=market,
+            accountOwner=self.public_address,
+            accountNumber=self.account_number,
+            transactionHash=None,
             limit=limit,
             startingBefore=startingBefore
         )
 
     def get_fills(
         self,
-        pairs,
-        makerAccountOwner=None,
-        makerAccountNumber=None,
+        market=None,
+        side=None,
+        accountOwner=None,
+        accountNumber=None,
+        transactionHash=None,
         limit=None,
-        startingBefore=None
+        startingBefore=None,
     ):
         '''
-        Return all historical fills
+        Returns all historical fills
 
-        :param pairs: required
-        :type pairs: list of str
+        :param market: optional
+        :type market: str[] of valid markets
+            ["WETH-DAI", "DAI-USDC", "WETH-USDC"]
+
+        :param side: optional
+        :type side: str in list ["BUY", "SELL"]
+
+        :param accountOwner: optional, defaults to self.public_address
+        :type accountOwner: str (address)
 
-        :param makerAccountOwner: optional
-        :type makerAccountOwner: str (address)
+        :param accountNumber: optional, defaults to self.account_number
```

Comment:
I don't think this default was implemented

Reply:
took out that line, looks like those `get_my_trades/orders/fills` fulfills that default parameter.